rust/feature-flags/src/config.rs

```diff
@@ -98,6 +98,12 @@ pub struct Config {
 
     #[envconfig(from = "TEAM_IDS_TO_TRACK", default = "all")]
     pub team_ids_to_track: TeamIdsToTrack,
+
+    #[envconfig(from = "CACHE_MAX_CAPACITY", default = "100000")]
+    pub cache_max_capacity: u64,
```

Comment:
nit: adding a unit to the name like `cache_ttl_seconds` has would be better? e.g. `cache_max_capacity_bytes` / `cache_max_bytes_capacity`

Reply:
v good point, will do that.